libs/template.js

```diff
@@ -1,14 +1,23 @@
 
-// generate a wrapper template
-const generate = (filename, module) => `const original = require('../${filename}');
-
-module.exports['handler'] = function (evt, ctx, cb) {
+const DEFAULT_WRAPPER = `const wrapper = (original) => (evt, ctx, cb) => {
```

Comment:
This should be a separate file. That would make it easy to do targeted testing and add additional validation that custom wrappers work by using the same code flow.

Reply:
That's a good idea